fiorella/src/styles/layout/slider.scss

```diff
@@ -0,0 +1,153 @@
+.slide {
+  display: none;
+}
+
+.slider-container {
+  position: relative;
+}
+
+.slider-navigator {
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  top: 10px;
+  width: 100%;
+}
+
+.slider-nav-button {
+  background-color: $color-white-solid;
+  border-radius: 50%;
+  border-style: none;
+  font-family: sans-serif;
+  margin: 0 5px;
+  height: 12px;
+  width: 5px;
+}
+
+.slider-nav-button-selected {
+  background-color: $color-featured;
+}
+
+.show-slide {
+  display: flex;
+}
+
+.slide {
+  flex-direction: column;
+  position: relative;
+
+  img {
+    width: 100%;
+  }
+}
+
+.slide-description {
+  bottom: 60px;
+  font-family: $font-slider-description;
+  font-style: italic;
+  font-size: 15px;
+  position: absolute;
+  right: 15px;
+  text-align: right;
+  width: 150px;
+}
+
+.slide-button {
+  display: inline;
```

Comment:
Properties should be ordered background-color, border-color, border-style, bottom, color, display, font-family, font-weight, padding, position, right, text-align, text-decoration, text-transform